librad/src/git/types/remote.rs

```diff
@@ -0,0 +1,120 @@
+// This file is part of radicle-link
+// <https://github.com/radicle-dev/radicle-link>
+//
+// Copyright (C) 2019-2020 The Radicle Team <dev@radicle.xyz>
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License version 3 or
+// later as published by the Free Software Foundation.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see <https://www.gnu.org/licenses/>.
+
+use std::path::{Path, PathBuf};
+
+use super::AsRefspec;
+
+pub struct Remote {
+    /// The file path to the git monorepo.
+    pub monorepo: PathBuf,
```

Comment:
This is actually out-of-date, as Kim pointed out that I was wrong here. It should be `Url`, and more specifically for us it will be a `LocalUrl`, which is of the form `rad://<namespace>.git`